Service/LeantimeHelper.php

```diff
@@ -0,0 +1,173 @@
+<?php
+
+namespace Modules\ItkIssueCreate\Service;
+
+use Exception;
+use GuzzleHttp\Client;
+use Symfony\Component\HttpFoundation\Request;
+use App\Conversation;
+use App\Thread;
+
+/**
+ * Helper for Leantime connection and creating Leantime ticket.
+ */
+final class LeantimeHelper
+{
+
+  /**
+   * Leantime Api path.
+   */
+    private const API_PATH_JSONRPC = '/api/jsonrpc/';
+
+  /**
+   * Status for the ticket that is created.
+   */
+    private const LEANTIME_TICKET_STATUS = '3';
+
+  /**
+   * Path to the ticket (without the id)
+   */
+    private const LEANTIME_TICKET_PATH = '/tickets/showKanban#/tickets/showTicket/';
```

Comment:
Shouldn't this be without "/tickets/showKanban" i.e. '/#/tickets/showTicket/'

Reply:
Fixed